system-addon/common/Reducers.jsm

```diff
@@ -289,6 +305,7 @@ function Sections(prevState = INITIAL_STATE.Sections, action) {
       return prevState.map(section => Object.assign({}, section,
         {rows: section.rows.filter(site => !action.data.includes(site.url))}));
     case at.PLACES_LINK_BLOCKED:
+    case at.REMOVE_FROM_POCKET:
       return prevState.map(section =>
         Object.assign({}, section, {rows: section.rows.filter(site => site.url !== action.data.url)}));
```

Comment:
Hmm.. Removing a specific pocket item isn't the same as blocking a url.. where this code would hide any other bookmarks of the same url when unpocketing.